release/ray_release/command_runner/_anyscale_job_wrapper.py

```diff
@@ -52,33 +54,56 @@ def exponential_backoff_retry(
             retry_delay_s *= 2
 
 
-def run_aws_cp(source: str, target: str):
+def install_aws_cli():
     global AWS_CLI_INSTALLED
+    if not AWS_CLI_INSTALLED:
+        subprocess.run(["pip", "install", "-q", "awscli"], check=True)
+        AWS_CLI_INSTALLED = True
+
+
+def install_gsutil_cli():
+    global GSUTIL_CLI_INSTALLED
+    if not GSUTIL_CLI_INSTALLED:
+        subprocess.run(["pip", "install", "-q", "gsutil"], check=True)
+        GSUTIL_CLI_INSTALLED = True
```

Comment:
nit: since we are already changing this, maybe make the `AWS_CLI_INSTALLED` into a set of `_aws_cli_install_once`, so that we can just set with `_aws_cli_install_once.add(True)` and check with `True in _aws_cli_install_once`, and do not need the `global`?

also not sure if this needs to concurrency safe or not.